### ISSUE TITLE
Tooltip text not working with AbstractTrees

### ISSUE THREAD

**Issue by ebalaban** [sisl/D3Trees.jl#16]
The problem is with lines 105-106 of [D3Trees.jl](https://github.com/sisl/D3Trees.jl/blob/master/src/D3Trees.jl). Once `take!(iob)` is called on line 105, `iob` is reset, so the second call on line 106 pushes an empty string into `t.tooltip`. For now, I simply fixed it on my end as
```julia
 str = String(take!(iob))
 push!(t.text, str)
 push!(t.tooltip, str)
```
Happy to create a pull request. It may be good to make this a bit more flexible down the road and allow different text for `t.text` and `t.tooltip`. If we stick with the same `AbstractTrees.printnode` interface, perhaps the two pieces of text can be separated by a delimiter within the input string (no delimiter: use the same text for both). In general, this tooltip feature is very useful for larger, denser trees.

**Comment by zsunberg** [comment]
Yes, please do make a PR! Thanks for pointing this out! - Zach

**Comment by ebalaban** [comment]
Ok, done. I have not migrated to 0.7/1.0 quite yet, so I just did it through GitHub (this little piece of code is common to 0.6 and 0.7/1.0).

**Comment by ebalaban** [comment]
Fixed with 2663457. Thanks @zsunberg!
